elao.app.docker/Makefile.dist

```diff
@@ -3,48 +3,94 @@
 -include .manala/Makefile
 
 define setup
-	$(DOCKER_MAKE) install build
+	$(DOCKER_MAKE) install db.install build
 endef
 
 ###########
 # Install #
 ###########
 
 ## Install application
-install:
-	# # Composer
-	# composer install --ansi --verbose
-	# # Doctrine
-	# bin/console doctrine:database:create --ansi --if-not-exists
-	# # Npm
+install: install.composer install.npm
+
+install.composer:
+	# symfony composer install --ansi --verbose
+
+install.npm:
 	# npm install --color=always
-	# # Yarn
-	# yarn install --color=always
 
 install@production: export APP_ENV = prod
 install@production:
 	# # Composer
 	# composer install --ansi --verbose --no-interaction --no-progress --prefer-dist --optimize-autoloader --no-scripts --no-dev
 	# # Npm
-	# npm install --color=always --no-progress --no-audit
-	# # Yarn
-	# yarn install --color=always --no-progress
+	# npm install --color=always --no-progress --no-audit --no-fund
 
 install@staging: export APP_ENV = prod
 install@staging:
 	# # Composer
 	# composer install --ansi --verbose --no-interaction --no-progress --prefer-dist --optimize-autoloader --no-scripts
 	# # Npm
-	# npm install --color=always --no-progress --no-audit
-	# # Yarn
-	# yarn install --color=always --no-progress
+	# npm install --color=always --no-progress --no-audit --no-fund
+
+############
+# Database #
+############
+
+## Database - Init (create+schema)
+db.install: db.create db.update-force
+
+## Database - Create (db only)
+db.create:
+	symfony console doctrine:database:create --if-not-exists
+
+# Database - Migrate (run migrations)
+db.migrate:
+	# If you use Messenger, without auto-setup nor a migration:
+	# Messenger:
+	# symfony console messenger:setup-transports
+	# Doctrine:
+	symfony console doctrine:migrations:sync-metadata-storage
+	symfony console doctrine:migration:migrate --no-interaction
+
+## Database - Generate a migration from current db schema diff
+db.diff:
+	symfony console doctrine:migrations:diff
+
+## Database - Update. Use FORCE=1 to force the schema update rather than using migrations
+db.update: export FORCE ?= 0
+db.update: $(if $(FORCE), db.update-force, db.migrate)
+
+# Database - Force update the database schema
+db.update-force: db.create
```

Comment:
Now hidden from help by default. Same for `db.migrate`. Main entrypoint `db.update` or `db.update FORCE=1`